### DISCUSSION TITLE
[V5] dark主题部分元素没有按预期切换

### DISCUSSION THREAD

**Discussion by recursively** [antvis/G6#6311] [Q&A] [answered]
如下所示：

<img width="711" alt="image" src="https://github.com/user-attachments/assets/ae53b106-a398-48f5-9dc9-a93ce1f5a3ce">

tooltip和minimap没有按照预期切换成dark主题，整体不太协调，这是设计之初就这样决定的吗？并且拿tooltip这种组件来讲，没有提供style来自定义样式，貌似只能自己去挨个改css，过于麻烦了

**Comment by Aarebecca** [comment, marked as answer]
tooltip 和 minimap 暂时还没有适配主题机制，后续我们会安排适配
